docs/pages/access-graph.mdx

```diff
@@ -0,0 +1,166 @@
+7. Resource Groups
+
+    ![Resource Group Node](../img/access-graph/resource-group-node.png)
+
+    Resource Groups are collections of resources. They can be used to organize
+    resources based on their role or team.
+
+8. Resources
+
+    ![Resource Node](../img/access-graph/resource-node.png)
+
+    Resources are the things that users can or cannot access. They can be
+    servers, databases, or Kubernetes clusters.
+
+## Graph paths
+
+Teleport Access Graph shows the relationships between users, roles, and
+resources. It does this by showing paths between nodes. Paths are the
+relationships between nodes. Paths always go from:
+
+  Users to User Groups to Actions to Resource Groups to Resources
+
+Paths can be divided into two categories:
+
+1. Allow paths
+
+    ![Allow Path](../img/access-graph/allow-path.png)
+
+    Allow paths connect users to resources. They show what a user can access
+    and what actions they can perform.
+
+2. Deny paths
+
+      ![Deny Path](../img/access-graph/deny-path.png)
+
+      Deny paths connect users to resources. They show what a user cannot access
+      and what actions they cannot perform. Deny paths take precedence over allow
+      paths.
+
+## How to use it
+
+Teleport Access Graph can help you to answer questions like:
+
+- Who can access a specific resource?
+
+![Show Access Path Resource](../img/access-graph/show-access-path-resource.gif)
+
+- What resources can a specific user access?
+
+![Show Access Path](../img/access-graph/show-access-path.gif)
+
+
+## How Teleport resources are represented
+
+Teleport Access Graph imports all Teleport resources and keeps them up to date, so every time you make a change
+to your Teleport resources, the Access Graph will reflect those changes.
+
+#### Users
+
+Users are created from Teleport Users.
+
+#### User Groups
+
+User Groups are created from Teleport Roles and access requests. Roles create a User Groups where the members
+are the users that have that role. Access requests create a temporary User Group where the members are the users that
+got the access through the accepted access request.
+
+#### Actions
+
+Actions are created from Teleport Roles. Actions can be divided into three
+categories:
+
+1. Allow Actions
+
+    Allow Actions are created from Teleport Roles. Allow Actions are the things
+    that users can do. For example, a user can SSH into a node.
+
+2. Deny Actions
+
+    Deny Actions are created from Teleport Roles. Deny Actions are the things
+    that users cannot do. For example, a user cannot SSH into a node. Deny
+    Actions take precedence over Allow Actions.
+
+3. Temporary Actions
+
+    Temporary Actions are created when a user is granted temporary access to a
+    resource. They are automatically deleted when the user's access expires.
+
+#### Resource Groups
+
+Resource Groups are created from Teleport Roles.
+
+#### Resources
+
+Resources are created from Teleport resources like nodes, databases, and
+Kubernetes clusters.
```

Comment:
```suggestion
Kubernetes clusters. Currently, only SSH nodes are supported.
```
Is this what the Admonition at the top means? If so, I've suggested mentioning it here as well.